### ISSUE TITLE
Links in message textbox have no contrast to background

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#251]
```
Pino 0.2.8 fixed the link color in the timeline (issue 195) with the new
Ubuntu themes.

But in the textbox for writing new tweets the link color is still mostly
unreadable because of low contrast.

It would be nice if pino could use the same link color as in the timeline
there.


(Tested with Pino 0.2.8 on Ubuntu 10.4)
```

Original issue reported on code.google.com by `m...@nilsbreyer.eu` on 9 May 2010 at 4:49

**Comment by GoogleCodeExporter** [comment]
```
Closed by a2f6568204 revision.
```

Original comment by `tro...@gmail.com` on 14 May 2010 at 11:12
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks very much for fixing this!

BTW, Pino is awesome :-)
```

Original comment by `m...@nilsbreyer.eu` on 15 May 2010 at 4:17
